debian/control

```diff
@@ -0,0 +1,21 @@
+Source: elnok
+Section: science
+Priority: optional
+Maintainer: Éric Piel <piel@delmic.com>
+Standards-Version: 4.1.4
+Build-Depends: debhelper (>= 11),
+                dh-python,
+                python3-setuptools,
+                python3-all-dev (>= 3.6.1)
+X-Python3-Version: >= 3.6
+Homepage: http://www.delmic.com
+Vcs-Browser: https://github.com/delmic/elnok
+
+Package: python3-elnok
```

Comment:
The package should be called `elnok`. Here, we provide a utility program. We don't care in which language this was written in. You have to add `python3-` for the packages which provide a python module.